src/commands/alias/set.ts

```diff
@@ -398,70 +399,18 @@ function handleCreateAliasError<T>(
     );
     return 1;
   }
-  if (error instanceof ERRORS.DomainConfigurationError) {
-    output.error(
-      `We couldn't verify the propagation of the DNS settings for ${chalk.underline(
-        error.meta.domain
-      )}`
-    );
-    if (error.meta.external) {
-      output.print(
-        `  The propagation may take a few minutes, but please verify your settings:\n\n`
-      );
-      output.print(
-        `${dnsTable([
-          error.meta.subdomain === null
-            ? ['', 'ALIAS', 'alias.zeit.co']
-            : [error.meta.subdomain, 'CNAME', 'alias.zeit.co']
-        ])}\n`
-      );
-    } else {
-      output.print(
-        `  We configured them for you, but the propagation may take a few minutes.\n`
-      );
-      output.print(`  Please try again later.\n`);
-    }
-    return 1;
-  }
-  if (error instanceof ERRORS.TooManyCertificates) {
-    output.error(
-      `Too many certificates already issued for exact set of domains: ${error.meta.domains.join(
-        ', '
-      )}`
-    );
-    return 1;
-  }
-  if (error instanceof ERRORS.CantSolveChallenge) {
-    if (error.meta.type === 'dns-01') {
-      output.error(
-        `The certificate provider could not resolve the DNS queries for ${
-          error.meta.domain
-        }.`
-      );
-      output.print(
-        `  This might happen to new domains or domains with recent DNS changes. Please retry later.\n`
-      );
-    } else {
-      output.error(
-        `The certificate provider could not resolve the HTTP queries for ${
-          error.meta.domain
-        }.`
-      );
-      output.print(
-        `  The DNS propagation may take a few minutes, please verify your settings:\n\n`
-      );
-      output.print(`${dnsTable([['', 'ALIAS', 'alias.zeit.co']])}\n`);
-    }
-    return 1;
-  }
-  if (error instanceof ERRORS.DomainValidationRunning) {
-    output.error(
-      `There is a validation in course for ${chalk.underline(
-        error.meta.domain
-      )}. Wait until it finishes.`
-    );
-    return 1;
+
+  if (
+    error instanceof ERRORS.CertError ||
+    error instanceof ERRORS.TooManyRequests ||
+    error instanceof ERRORS.DomainValidationRunning ||
+    error instanceof ERRORS.CertConfigurationError ||
+    error instanceof ERRORS.DomainNotFound ||
+    error instanceof ERRORS.CertsDNSError
+  ) {
+    return handleCertError(output, error);
```

Comment:
Maybe it would make more sense to have something like this:

```javascript
const certError = handleCertError(output, error);

if (certError) {
    return certError;
}

return error;
```

Since we might end up having another generic function for a specific field of errors and then we'd want to chain it